setup.py

```diff
@@ -50,6 +50,6 @@
     extras_require={
         'dev': ['flake8', 'mypy'],
         'unit-test': ['pytest', 'pytest-cov'],
-        'system-test': ['systest']
+        'system-test': ['systest', ]
```

Comment:
What is the reason for the addition of a spurious comma? Is it something which your editor insists on?

Reply:
No - it's because whilst trying to understand the issues with `cpp` output I thought I might be able to control installing a specific `gcc` version here, and when that didn't work I failed to tidy up after myself fully!